sdk/resourcemanager/azure-resourcemanager-appservice/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Features Added
 
 - Supported disabling public network access in `WebApp` via `disablePublicNetworkAccess()`, for private link feature.
+- Supported disabling public network access in `FunctionApp` via `disablePublicNetworkAccess()`, for private link feature.
+- Supported disabling public network access in `DeploymentSlot` via `disablePublicNetworkAccess()`, for private link feature.
+- Supported disabling public network access in `FunctionDeploymentSlot` via `disablePublicNetworkAccess()`, for private link feature.
```

Comment:
Seems we added support for snaptshot, search and sql as well. We should reflect them in the changelog.

Reply:
Never mind... I saw them in their changelogs.